higgstautau/lephad/objects.py

```diff
@@ -0,0 +1,70 @@
+from ..mixins import *
```

Comment:
@vincecr0ft We probably could live with a single of those files for the 3 channels